apps/scan/backend/test/helpers/custom_helpers.ts

```diff
@@ -256,7 +257,5 @@ export async function scanBallot(
     canUnconfigure: true,
   });
 
-  if (!options.skipWaitForContinuousExportToUsbDrive) {
-    await waitForContinuousExportToUsbDrive();
-  }
+  await waitForContinuousExportToUsbDrive(mockUsbDrive);
```

Comment:
Same idea here as above - we can perform this wait more liberally now

Note that I'm still including this wait here in the `scanBallot` helper, in spite of the wait at the end of `withApp`, because some tests explicitly interact with the exported contents on the USB drive. So a wait is needed right away as opposed to at the end of the test.